base/linalg/arnoldi.jl

```diff
@@ -237,7 +237,7 @@ function _eigs(A, B;
     end
 
     # Refer to ex-*.doc files in ARPACK/DOCUMENTS for calling sequence
-    matvecA(x) = A * x
+    matvecA!(y, x) = A_mul_B!(y, A, x)
     if !isgeneral           # Standard problem
         matvecB = x -> x
```

Comment:
Can't `matvecB` be changed to in-place as well, i.e. `copy!` here and `A_mul_B!` below?